ts/src/ellipx.ts

```diff
@@ -0,0 +1,1876 @@
+        //     ],
+        //     "paging": {
+        //         "page_no": 1,
+        //         "count": "1",
+        //         "page_max": 1,
+        //         "results_per_page": 20
+        //     }
+        // }
+        const data = this.safeValue (response, 'data', []);
+        return this.parseOrders (data, market, since, limit);
+    }
+
+    /**
+     * @method
+     * @name ellipx#fetchOrders
+     * @description fetches information on multiple orders made by the user
+     * @see https://docs.google.com/document/d/1ZXzTQYffKE_EglTaKptxGQERRnunuLHEMmar7VC9syM/edit?tab=t.0#heading=h.5z2nh2b5s81n
+     * @param {string} symbol unified market symbol of the market orders were made in
+     * @param {int|undefined} since timestamp in ms of the earliest order
+     * @param {int|undefined} limit the maximum amount of orders to fetch
+     * @param {object} params extra parameters specific to the exchange API endpoint
+     * @returns {Order[]} a list of [order structures]{@link https://docs.ccxt.com/#/?id=order-structure}
+     */
+    async fetchOrders (symbol: Str = undefined, since: Int = undefined, limit: Int = undefined, params = {}) {
+        return await this.fetchOrdersByStatus (undefined, symbol, since, limit, params);
+    }
+
+    /**
+     * @method
+     * @name ellipx#fetchOpenOrders
+     * @description fetches information on open orders made by the user
+     * @see https://docs.google.com/document/d/1ZXzTQYffKE_EglTaKptxGQERRnunuLHEMmar7VC9syM/edit?tab=t.0#heading=h.5z2nh2b5s81n
+     * @param {string} symbol unified market symbol of the market orders were made in
+     * @param {int|undefined} since timestamp in ms of the earliest order
+     * @param {int|undefined} limit the maximum amount of orders to fetch
+     * @param {object} params extra parameters specific to the exchange API endpoint
+     * @returns {Order[]} a list of [order structures]{@link https://docs.ccxt.com/#/?id=order-structure}
+     */
+    async fetchOpenOrders (symbol: Str = undefined, since: Int = undefined, limit: Int = undefined, params = {}): Promise<Order[]> {
+        return await this.fetchOrdersByStatus ('open', symbol, since, limit, params);
+    }
+
+    parseOrder (order, market = undefined): Order {
+        const id = this.safeString (order, 'Market_Order__');
+        const timestamp = this.safeInteger (this.safeDict (order, 'Created'), 'unixms');
+        const typeResponse = this.safeString (order, 'Type');
+        let side = 'sell';
+        if (typeResponse === 'bid') {
+            side = 'buy';
+        }
+        const status = this.parseOrderStatus (this.safeString (order, 'Status'));
+        const amount = this.parseAmount (this.safeDict (order, 'Amount'));
+        const price = this.parseAmount (this.safeDict (order, 'Price'));
+        const type = (price === undefined) ? 'market' : 'limit';
+        const executed = this.parseAmount (this.safeDict (order, 'Executed'));
+        const filled = executed;
```

Comment:
@nguyenhieuec can you check this? this logic seems off since I'm getting a negative remaining 

![image](https://github.com/user-attachments/assets/6dff391a-72d3-4551-8cc1-43f292c99423)

Reply:
wrong placement of amount and fill. I'll test and revise it today.

Reply:
I added new commit to resolve this. @carlosmiei.

Reply:
@nguyenhieuec thanks, will take a look

Reply:
@nguyenhieuec I think this is still wrong, check this out, I have bought 10 doge, and now amount is 20

```
2024-11-24T15:38:40.056Z iteration 0 passed in 590 ms

{
  id: 'mktor-hqufk4-emrb-gk5o-jkes-mqk4gqnu',
  clientOrderId: undefined,
  },
  timestamp: 1732369899616,
  datetime: '2024-11-23T13:51:39.616Z',
  lastTradeTimestamp: 1732369899616,
  status: 'closed',
  symbol: undefined,
  type: 'market',
  timeInForce: 'GTC',
  postOnly: false,
  side: 'buy',
  price: 0.471108,
  stopPrice: undefined,
  triggerPrice: undefined,
  average: 0.471108,
  cost: 4.71108,
  amount: 20,
  filled: 10,
  remaining: 10,
  fee: undefined,
```

Reply:
Hi, I have reviewed the issue.
I executed 2 buy orders and 2 sell orders and fetched the balance before that. However, I noticed that your `status: 'closed`' is different from my call, where the status should be `pending` when creating an order. Could you clarify which function is responsible for producing this status? Let me check other functions.
![fetchBalance then createOrder buy 1](https://github.com/user-attachments/assets/21a8d1e1-b96a-489d-bb1e-4ac87bfc411d)
![fetchBalance then createOrder buy 2](https://github.com/user-attachments/assets/3e3f5100-4db9-4a10-8e09-9cfc5a29fab1)
![fetchBalance then createOrder sell 1](https://github.com/user-attachments/assets/9a5bb61b-d14d-45db-b890-841a42be53e4)
![fetchBalance then createOrder sell 2](https://github.com/user-attachments/assets/3d123a04-e9ce-4420-a28c-7a7238adc2fc)

Reply:
I think the `safeOrder` change the default amount I parsed, my solution is add `parseNumber` and remove `remaining` logic keep the parsed info from api only.

Reply:
@carlosmiei I added a commit to resolve this comment. In addition, I want to ask about a `NullableDict` that was recently added to the code and makes the CI fail. Any things I can help with? Try to change to `safeDict` someplace but it could not pass a local test.

Reply:
@nguyenhieuec I will check this issue don't worry